hack/make-rules/test-cmd-util.sh

```diff
@@ -3007,10 +3007,10 @@ run_daemonset_history_tests() {
   # Command
   # Create a DaemonSet (revision 1)
   kubectl apply -f hack/testdata/rollingupdate-daemonset.yaml --record "${kube_flags[@]}"
-  kube::test::get_object_assert controllerrevisions "{{range.items}}{{$annotations_field}}:{{end}}" ".*rollingupdate-daemonset.yaml --record.*"
+  kube::test::wait_object_assert controllerrevisions "{{range.items}}{{$annotations_field}}:{{end}}" ".*rollingupdate-daemonset.yaml --record.*"
```

Comment:
@janetkuo We have to change `get_object_assert` to `wait_object_assert` here. :)